05/09/2016 20:30:15:277 ERROR - In Progress | session id:MPP-2016249123015-9 | CQGas:Welcome |  | com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
 | Exception stack trace | Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted 
 EXCEPTION> 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | 10000000341473078615
05/09/2016 20:30:15:378 ERROR - In Progress | session id:MPP-2016249123015-9 | CQGas:Welcome |  | com.avaya.sce.runtimecommon.SCERuntimeException: Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

**** Root Cause 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | Exception stack trace | Error processing request
 EXCEPTION> 
com.avaya.sce.runtimecommon.SCERuntimeException: Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

**** Root Cause 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:582)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	... 17 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | 10000000341473078615
05/09/2016 20:38:15:126 ERROR - In Progress | session id:MPP-2016249123815-8 | CQGas:Welcome |  | com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
 | Exception stack trace | Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted 
 EXCEPTION> 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | 10000000351473079095
05/09/2016 20:38:15:250 ERROR - In Progress | session id:MPP-2016249123815-8 | CQGas:Welcome |  | com.avaya.sce.runtimecommon.SCERuntimeException: Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

**** Root Cause 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | Exception stack trace | Error processing request
 EXCEPTION> 
com.avaya.sce.runtimecommon.SCERuntimeException: Error loading PluggableConnectorInitialize class [com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit] for initialVariablesPosted com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

**** Root Cause 
com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:582)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)
Caused by: com.avaya.sce.runtimecommon.SCERuntimeException: Unable to get socket connection: java.net.ConnectException: Connection refused: connect: Connection refused: connectjava.net.ConnectException: Connection refused: connect
**** Root Cause 
java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	at com.avaya.sce.runtime.AppDocument.processRequest(AppDocument.java:188)
	at com.avaya.sce.runtime.SCEServlet.requestHandler(SCEServlet.java:285)
	at com.avaya.sce.runtime.SCEServlet.doPost(SCEServlet.java:190)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:637)
	at javax.servlet.http.HttpServlet.service(HttpServlet.java:717)
	at org.apache.catalina.core.ApplicationFilterChain.internalDoFilter(ApplicationFilterChain.java:290)
	at org.apache.catalina.core.ApplicationFilterChain.doFilter(ApplicationFilterChain.java:206)
	at org.apache.catalina.core.StandardWrapperValve.invoke(StandardWrapperValve.java:233)
	at org.apache.catalina.core.StandardContextValve.invoke(StandardContextValve.java:191)
	at org.apache.catalina.core.StandardHostValve.invoke(StandardHostValve.java:128)
	at org.apache.catalina.valves.ErrorReportValve.invoke(ErrorReportValve.java:102)
	at org.apache.catalina.core.StandardEngineValve.invoke(StandardEngineValve.java:109)
	at org.apache.catalina.connector.CoyoteAdapter.service(CoyoteAdapter.java:286)
	at org.apache.coyote.http11.Http11Processor.process(Http11Processor.java:845)
	at org.apache.coyote.http11.Http11Protocol$Http11ConnectionHandler.process(Http11Protocol.java:583)
	at org.apache.tomcat.util.net.JIoEndpoint$Worker.run(JIoEndpoint.java:447)
	at java.lang.Thread.run(Unknown Source)

	at com.avaya.sce.runtimecommon.SCESession.throwRTException(SCESession.java:2301)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:182)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.<init>(CTICallOperation.java:62)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CallInfoInitialCall.<init>(CallInfoInitialCall.java:18)
	at com.avaya.sce.runtime.connectivity.cti.CTIOpFactory.createCallInfoInitialCallOperation(CTIOpFactory.java:48)
	at com.avaya.sce.runtime.connectivity.cti.pdc.CTIConnectorInit.initialVariablesPosted(CTIConnectorInit.java:91)
	at com.avaya.sce.runtime.SCEServlet.processRequest(SCEServlet.java:577)
	... 17 more
Caused by: java.net.ConnectException: Connection refused: connect
	at java.net.DualStackPlainSocketImpl.connect0(Native Method)
	at java.net.DualStackPlainSocketImpl.socketConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.doConnect(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connectToAddress(Unknown Source)
	at java.net.AbstractPlainSocketImpl.connect(Unknown Source)
	at java.net.PlainSocketImpl.connect(Unknown Source)
	at java.net.SocksSocketImpl.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.connect(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at java.net.Socket.<init>(Unknown Source)
	at com.avaya.sce.runtime.connectivity.cti.desktop.CTICallOperation.initComm(CTICallOperation.java:178)
	... 22 more
 | 10000000351473079095
